Testreglar/3.3.2/Nett/nett-3.3.2a.json

```diff
@@ -0,0 +1,488 @@
+							"verdi": "Nei",
+							"handling": {
+								"type": "avslutt",
+								"fasit": "Nei",
+								"utfall": "Inndataelement har identifikasjon i form av tekst eller ikon/symbol/bilde som ikkje alltid er synleg når inndataelementet er i fokus."
+							}
+						},
+						"2": {
+							"sjekk": "3.4",
+							"type": "ulik",
+							"verdi": "Nei",
+							"handling": {
+								"type": "avslutt",
+								"fasit": "Nei",
+								"utfall": "Inndataelement har identifikasjon i form av: #delutfall(0,Nei)<br>- ikon/symbol/bilde, som ikkje alltid er synlege når inndataelementet er i fokus"
+							}
+						}
+					}
+				}
+			}
+		},
+		{
+			"stegnr": "3.8",
+			"spm": "Har ikonet/symbolet/bildet tekstalternativ, som set brukaren i stand til å identifisere inndataelementet?",
+			"ht": "<p>Slik testar du:</p>\n<ul>\n<li>Inspiser ikonet/symbolet/bildet og bruk Accessibility Checker i nettlesaren</li>\n<li>Vurder om ikonet/symbolet/bildet har tekstalternativ i koden. Prioritert rekkefølgje er:\n<ol>\n<li><code>aria-label</code></li>\n<li><code>aria-labelledby</code></li>\n<li><code>alt</code>, finst helst på <code>&lt;img&gt;</code>, men kan og brukast på andre</li>\n<li><code>value</code> ,finst helst på skjemaelement</li>\n<li>tekst i ikon/symbol-elementet, for eksempel <code>&lt;button&gt;Søk&lt;/button&gt;</code>)</li>\n<li><code>title</code></li>\n</ol>\n</li>\n<li>Du skal vurdere om innhaldet i attributta set brukaren i stand til å identifisere inndataelementet.</li>\n</ul>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "regler",
+					"regler": {
+						"1": {
+							"sjekk": "2.5",
+							"type": "lik",
+							"verdi": "Nei",
+							"handling": {
+								"type": "gaaTil",
+								"steg": "3.9",
+								"delutfall": {
+									"nr": 1,
+									"tekst": "identifikasjon i form av ikon/symbol/bilde",
+									"fasit": "Ja"
+								}
+							}
+						},
+						"2": {
+							"sjekk": "2.5",
+							"type": "lik",
+							"verdi": "Ja",
+							"handling": {
+								"type": "avslutt",
+								"fasit": "Ja",
+								"utfall": "Inndataelement har identifikasjon i form av ikon/symbol/bilete. Skjema har tekstleg informasjon om at alle inndataelementer er obligatoriske."
+							}
+						}
+					}
+				},
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Nei",
+					"utfall": "Inndataelement med identifikasjon i form av: #delutfall(0,Nei)<br>- ikon/symbol/bilde, som ikkje er identifisert i koden."
+				}
+			}
+		},
+		{
+			"stegnr": "3.9",
+			"spm": "Er det aktuelle inndataelementet obligatorisk?",
+			"ht": "<p>Obligatorisk inneber at brukaren må fylle ut informasjon eller gjere valg, for å kome vidare i skjemaet, for eksempel</p>\n<ul>\n<li>fylle ut informasjon </li>\n<li>gjere val i nedtrekksmenyar, radioknappar, avkryssingsboksar.</li>\n</ul>\n<p><strong>Merk:</strong> Du må kanskje fylle ut eller prøve å sende inn skjemaet for å finne ut om inndataelementet er obligatorisk.</p>",
```

Comment:
Hva med ferdigutfylt tekst i skjemaelement som er obligatorisk?

Er det obligatorisk hvis du ikke får sendt inn skjema uten å først skriv noe inn? Knapp er disabled til det er innhold? Isåfall må vi ha noe informasjon om hvordan det kan sjekkes siden du ikke får sende inn skjema da. Kan være lurt å sjekke kode for required o.l?

Reply:
Har lagt inn punktlista og sjekke koden som alternativ.